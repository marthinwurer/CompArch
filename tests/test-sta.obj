100 1 00001
101 1 00005
102 1 fffff
103 1 ffffb
104 1 00000
105 1 00000
300 1 10100
301 1 12104
302 1 10105
303 1 10104
304 1 10101
305 1 12104
306 1 10105
307 1 10104
308 1 10102
309 1 12104
30a 1 10105
30b 1 10104
30c 1 10103
30d 1 12104
30e 1 10105
30f 1 10104
310 1 00000
300
