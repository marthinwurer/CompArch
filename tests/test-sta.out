CPU "ARCH" Simulator, 2.5a(Jan 22 2015)
-----------------------------------------

Memory sets starting address to 300
300:  10100  LDA   100  A[00000]  B[00000]  XR[000]
301:  12104  STA   104  A[00001]  B[00000]  XR[000]
302:  10105  LDA   105  A[00001]  B[00000]  XR[000]
303:  10104  LDA   104  A[00000]  B[00000]  XR[000]
304:  10101  LDA   101  A[00001]  B[00000]  XR[000]
305:  12104  STA   104  A[00005]  B[00000]  XR[000]
306:  10105  LDA   105  A[00005]  B[00000]  XR[000]
307:  10104  LDA   104  A[00000]  B[00000]  XR[000]
308:  10102  LDA   102  A[00005]  B[00000]  XR[000]
309:  12104  STA   104  A[fffff]  B[00000]  XR[000]
30a:  10105  LDA   105  A[fffff]  B[00000]  XR[000]
30b:  10104  LDA   104  A[00000]  B[00000]  XR[000]
30c:  10103  LDA   103  A[fffff]  B[00000]  XR[000]
30d:  12104  STA   104  A[ffffb]  B[00000]  XR[000]
30e:  10105  LDA   105  A[ffffb]  B[00000]  XR[000]
30f:  10104  LDA   104  A[00000]  B[00000]  XR[000]
310:  00000  HALT       A[ffffb]  B[00000]  XR[000]

Machine Halted - HALT instruction executed

Simulated time 100 cycles

LAST CPUObject DESTROYED; END OF SIMULATION
